doc/getting-started/rt_industry.rst

```diff
@@ -0,0 +1,270 @@
+   |                      |                   +----------------------+-----------------------------------------------------------+
+   |                      |                   | Storage capabilities | - SATA: 1TB WDC WD10SPZX-22Z                              |
+   |                      |                   |                      | - NVMe: 256G Intel Corporation SSD Pro 7600p/760p/E 6100p |
+   +----------------------+-------------------+----------------------+-----------------------------------------------------------+
+
+How to set up ACRN Hypervisor for industry scenario
+***************************************************
+There are several ways to set up ACRN industry scenario environment. Two of them listed below
+are recommended:
+
+- :ref:`Using the pre-installed industry ACRN hypervisor <use pre-installed industry efi>`
+- :ref:`Using the ACRN industry out-of-the-box image <use industry ootb image>`
+
+.. _use pre-installed industry efi:
+
+Using the pre-installed industry ACRN hypervisor
+================================================
+Firstly we need to follow :ref:`ACRN quick setup guide <quick-setup-guide>` to set up
+ACRN Service VM. The industry hypervisor image is installed in ``/usr/lib/acrn/``
+directory once the Service VM boots. You may simply follow these steps to use
+``acrn.kbl-nuc-i7.industry.efi`` instead of the original SDC hypervisor:
+
+.. code-block:: none
+
+   $ sudo mount /dev/sda1 /mnt
+   $ sudo mv /mnt/EFI/acrn/acrn.efi /mnt/EFI/acrn/acrn.efi.bak
+   $ sudo cp /usr/lib/acrn/acrn.kbl-nuc-i7.industry.efi /mnt/EFI/acrn/acrn.efi
+   $ sync && umount /mnt
+   $ sudo reboot
+
+.. note:: Skip this section if you choose following the :ref:`Using the ACRN industry out-of-the-box image <use industry ootb image>`
+
+.. _use industry ootb image:
+
+Using the ACRN industry out-of-the-box image
+============================================
+#. Download the
+   `sos-industry-31090.img.xz <https://github.com/projectacrn/acrn-hypervisor/releases/download/acrn-2019w39.1-140000p/sos-industry-31090.img.xz>`_
+   to your development machine.
+#. Decompress the xz image
+
+  .. code-block:: none
+
+     $ xz -d sos-industry-31090.img.xz
+
+#. Follow the
+   `Deploy the Service VM image <https://projectacrn.github.io/latest/getting-started/acrn_ootb.html#step-3-deploy-the-service-vm-image>`_
+   to deploy the Service VM image to the SATA disk.
+
+Install and launch Preempt-RT VM
+********************************
+#. Download the
+   `preempt-rt-31090.img.xz <https://github.com/projectacrn/acrn-hypervisor/releases/download/acrn-2019w39.1-140000p/preempt-rt-31090.img.xz>`_
+   to your development machine.
+#. Decompress the xz image
+
+   .. code-block:: none
+
+      $ xz -d preempt-rt-31090.img.xz
+#. Follow the
+   `Deploy the User VM Preempt-RT image <https://projectacrn.github.io/latest/getting-started/acrn_ootb.html#step-3-deploy-the-user-vm-preempt-rt-image>`_
```

Comment:
Same comment re: using a ReST hyperlink instead of URL

Reply:
Done